importmap.php

```diff
@@ -45,4 +45,12 @@
     '@symfony/ux-live-component' => [
         'path' => './vendor/symfony/ux-live-component/assets/dist/live_controller.js',
     ],
+    'tom-select/dist/css/tom-select.bootstrap4.css' => [
```

Comment:
Do you need both?

Reply:
that's automatical now, after the composer install.